bin/console

```diff
@@ -39,10 +39,6 @@ if ($_SERVER['APP_DEBUG']) {
     }
 }
 
-if (isset($_ENV['BUNDLES_TO_ENABLE'])) {
-    $_SERVER['BUNDLES_TO_ENABLE'] = $_ENV['BUNDLES_TO_ENABLE'];
-}
```

Comment:
Why are you removing that? I would be for leaving the earlier behaviour with using this env.